### ISSUE TITLE
Crash by: KAI-PC\Kai at 22.06.2024+19:50:32

### ISSUE THREAD

**Issue by FPGSchiba** [FPGSchiba/VNGD-SimpleRadioStandalone#129]
Crash Log:
```
System.IO.FileNotFoundException: Die Datei "E:\ExpandIcon.png" konnte nicht gefunden werden.
Dateiname: "E:\ExpandIcon.png"
   bei System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   bei System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   bei System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share)
   bei System.Windows.Media.Imaging.BitmapDecoder.SetupDecoderFromUriOrStream(Uri uri, Stream stream, BitmapCacheOption cacheOption, Guid& clsId, Boolean& isOriginalWritable, Stream& uriStream, UnmanagedMemoryStream& unmanagedMemoryStream, SafeFileHandle& safeFilehandle)
   bei System.Windows.Media.Imaging.BitmapDecoder.CreateFromUriOrStream(Uri baseUri, Uri uri, Stream stream, BitmapCreateOptions createOptions, BitmapCacheOption cacheOption, RequestCachePolicy uriCachePolicy, Boolean insertInDecoderCache)
   bei System.Windows.Media.Imaging.BitmapImage.FinalizeCreation()
   bei System.Windows.Media.Imaging.BitmapSource.CompleteDelayedCreation()
   bei System.Windows.Media.Imaging.BitmapSource.get_WicSourceHandle()
   bei System.Windows.Media.Imaging.BitmapSource.get_DUCECompatiblePtr()
   bei System.Windows.Media.Imaging.BitmapSource.UpdateBitmapSourceResource(Channel channel, Boolean skipOnChannelCheck)
   bei System.Windows.Media.Imaging.BitmapSource.AddRefOnChannelCore(Channel channel)
   bei System.Windows.Media.Imaging.BitmapSource.System.Windows.Media.Composition.DUCE.IResource.AddRefOnChannel(Channel channel)
   bei System.Windows.Media.ImageBrush.AddRefOnChannelCore(Channel channel)
   bei System.Windows.Media.Brush.System.Windows.Media.Composition.DUCE.IResource.AddRefOnChannel(Channel channel)
   bei System.Windows.Media.RenderData.System.Windows.Media.Composition.DUCE.IResource.AddRefOnChannel(Channel channel)
   bei System.Windows.UIElement.RenderContent(RenderContext ctx, Boolean isOnChannel)
   bei System.Windows.Media.Visual.RenderRecursive(RenderContext ctx)
   bei System.Windows.Media.Visual.UpdateChildren(RenderContext ctx, ResourceHandle handle)
   bei System.Windows.Media.Visual.RenderRecursive(RenderContext ctx)
   bei System.Windows.Media.Visual.UpdateChildren(RenderContext ctx, ResourceHandle handle)
   bei System.Windows.Media.Visual.RenderRecursive(RenderContext ctx)
   bei System.Windows.Media.Visual.UpdateChildren(RenderContext ctx, ResourceHandle handle)
   bei System.Windows.Media.Visual.RenderRecursive(RenderContext ctx)
   bei System.Windows.Media.Visual.UpdateChildren(RenderContext ctx, ResourceHandle handle)
   bei System.Windows.Media.Visual.RenderRecursive(RenderContext ctx)
   bei System.Windows.Media.Visual.UpdateChildren(RenderContext ctx, ResourceHandle handle)
   bei System.Windows.Media.Visual.RenderRecursive(RenderContext ctx)
   bei System.Windows.Media.Visual.UpdateChildren(RenderContext ctx, ResourceHandle handle)
   bei System.Windows.Media.Visual.RenderRecursive(RenderContext ctx)
   bei System.Windows.Media.Visual.UpdateChildren(RenderContext ctx, ResourceHandle handle)
   bei System.Windows.Media.Visual.RenderRecursive(RenderContext ctx)
   bei System.Windows.Media.Visual.UpdateChildren(RenderContext ctx, ResourceHandle handle)
   bei System.Windows.Media.Visual.RenderRecursive(RenderContext ctx)
   bei System.Windows.Media.Visual.UpdateChildren(RenderContext ctx, ResourceHandle handle)
   bei System.Windows.Media.Visual.RenderRecursive(RenderContext ctx)
   bei System.Windows.Media.Visual.UpdateChildren(RenderContext ctx, ResourceHandle handle)
   bei System.Windows.Media.Visual.RenderRecursive(RenderContext ctx)
   bei System.Windows.Media.Visual.UpdateChildren(RenderContext ctx, ResourceHandle handle)
   bei System.Windows.Media.Visual.RenderRecursive(RenderContext ctx)
   bei System.Windows.Media.Visual.UpdateChildren(RenderContext ctx, ResourceHandle handle)
   bei System.Windows.Media.Visual.RenderRecursive(RenderContext ctx)
   bei System.Windows.Media.Visual.UpdateChildren(RenderContext ctx, ResourceHandle handle)
   bei System.Windows.Media.Visual.RenderRecursive(RenderContext ctx)
   bei System.Windows.Media.Visual.UpdateChildren(RenderContext ctx, ResourceHandle handle)
   bei System.Windows.Media.Visual.RenderRecursive(RenderContext ctx)
   bei System.Windows.Media.Visual.Render(RenderContext ctx, UInt32 childIndex)
   bei System.Windows.Media.CompositionTarget.Compile(Channel channel)
   bei System.Windows.Media.CompositionTarget.System.Windows.Media.ICompositionTarget.Render(Boolean inResize, Channel channel)
   bei System.Windows.Media.MediaContext.Render(ICompositionTarget resizedCompositionTarget)
   bei System.Windows.Media.MediaContext.RenderMessageHandlerCore(Object resizedCompositionTarget)
   bei System.Windows.Media.MediaContext.RenderMessageHandler(Object resizedCompositionTarget)
   bei System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   bei System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
   bei System.Windows.Threading.DispatcherOperation.InvokeImpl()
   bei MS.Internal.CulturePreservingExecutionContext.CallbackWrapper(Object obj)
   bei System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   bei System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
   bei System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state)
   bei MS.Internal.CulturePreservingExecutionContext.Run(CulturePreservingExecutionContext executionContext, ContextCallback callback, Object state)
   bei System.Windows.Threading.DispatcherOperation.Invoke()
   bei System.Windows.Threading.Dispatcher.ProcessQueue()
   bei System.Windows.Threading.Dispatcher.WndProcHook(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
   bei MS.Win32.HwndWrapper.WndProc(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
   bei MS.Win32.HwndSubclass.DispatcherCallbackOperation(Object o)
   bei System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   bei System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
   bei System.Windows.Threading.Dispatcher.LegacyInvokeImpl(DispatcherPriority priority, TimeSpan timeout, Delegate method, Object args, Int32 numArgs)
   bei MS.Win32.HwndSubclass.SubclassWndProc(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam)
   bei MS.Win32.UnsafeNativeMethods.DispatchMessage(MSG& msg)
   bei System.Windows.Threading.Dispatcher.PushFrameImpl(DispatcherFrame frame)
   bei System.Windows.Application.RunDispatcher(Object ignore)
   bei System.Windows.Application.RunInternal(Window window)
   bei DCS_SR_Client.App.Main()
```

This is an automatic message and is only here so that the Development Team can work fast.

**Comment by FPGSchiba** [comment]
This has to do with the relative Paths and maybe copy the `.exe` out of the directory. We will need to embed the Icon into the XAML and reference as a Project resource. 

Files: `RadioOverlayTransparent.xaml.cs` Line: `329`

**Comment by FPGSchiba** [comment]
Fixed with 1.3.2
